packages/studio-base/src/panels/NodePlayground/BottomBar/index.tsx

```diff
@@ -132,7 +121,7 @@ const BottomBar = ({ nodeId, isSaved, save, diagnostics, logs }: Props): ReactEl
               }
               value="diagnostics"
               data-testid="np-errors"
-              onClick={() => handleClick("diagnostics")}
+              onClick={() => handleClick()}
```

Comment:
nit: 
```suggestion
              onClick={handleClick} 
```
(or alternatively `onChangeTab` since handleClick just wraps that)
and below